common/src/main/java/com/wynntils/features/user/ChatTabsFeature.java

```diff
@@ -29,19 +31,20 @@
 import net.minecraft.client.gui.screens.ChatScreen;
 import net.minecraftforge.eventbus.api.EventPriority;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
 
 public class ChatTabsFeature extends UserFeature {
     public static ChatTabsFeature INSTANCE;
 
     @Config(visible = false)
     public List<ChatTab> chatTabs = Arrays.asList(
-            new ChatTab("All", false, null, null),
-            new ChatTab("Global", false, Sets.newHashSet(RecipientType.GLOBAL), null),
-            new ChatTab("Local", false, Sets.newHashSet(RecipientType.LOCAL), null),
-            new ChatTab("Guild", false, Sets.newHashSet(RecipientType.GUILD), null),
-            new ChatTab("Party", false, Sets.newHashSet(RecipientType.PARTY), null),
-            new ChatTab("Private", false, Sets.newHashSet(RecipientType.PRIVATE), null),
-            new ChatTab("Shout", false, Sets.newHashSet(RecipientType.SHOUT), null));
+            new ChatTab("All", false, null, null, null),
+            new ChatTab("Global", false, null, Sets.newHashSet(RecipientType.GLOBAL), null),
+            new ChatTab("Local", false, null, Sets.newHashSet(RecipientType.LOCAL), null),
+            new ChatTab("Guild", false, "/g ", Sets.newHashSet(RecipientType.GUILD), null),
+            new ChatTab("Party", false, "/p ", Sets.newHashSet(RecipientType.PARTY), null),
+            new ChatTab("Private", false, "/msg ", Sets.newHashSet(RecipientType.PRIVATE), null),
```

Comment:
Is there not a `/r` (for "reply") that let's you answer the last private message? Perhaps that would be a better auto message for "Private"?

Reply:
I can see both msg and r being useful

Reply:
Yes, but we can reasonably only select one. I would argue for `/r`, since it is a reasonable default, and it is easier to backspace a single `r` if you want to change it to `msg`. If you default to `/msg`, you'll either have to delete `msg` to type `r`, or type the name of the person you want to answer.

Reply:
Changed it.